spec/models/survey_spec.rb

```diff
@@ -4,4 +4,28 @@
   it "has valid factory" do
     expect(create(:survey)).to be_valid
   end
+
+  context "associations" do
+    it { is_expected.to have_many(:questions).class_name("SurveyQuestion") }
+    it { is_expected.to have_many(:responses).class_name("SurveyResponse") }
+  end
+
+  context "validations" do
+    it { is_expected.to validate_presence_of(:name) }
+  end
+
+  context "enum" do
+    it { is_expected.to define_enum_for(:status).with_values(Inactive: 0, Active: 1) }
+  end
+
+  describe ".complete" do
+    it "includes completed surveys" do
+      survey = create(:survey, status: 0)
+      expect(Survey.complete).to include(survey)
```

Comment:
Hey @garettarrowood! While @jelenad1 and I were working on this we came across a possible bug. The scope on `survey` is currently set up like: 

<img width="634" alt="survey_rb" src="https://user-images.githubusercontent.com/17501514/139496915-3701c526-c5a9-47b2-8e69-38eb54ff110e.png">


so it seems this scope will not function properly until either it, or the enum, are changed. We thought about updating the enum to be `{ complete: 0, incomplete: 1 }` since a survey currently defaults to `status: 1`, however that is the opposite of other models, such as `SurveyResponse`. Since we're fairly new to the app, we wanted to run it by you for input before we made any major changes.

Reply:
Yep! That's a mistake. Let's define the status as `{ complete: 0, incomplete: 1 }` because it looks like the db is defaulting this column to 1.

Reply:
Yep! This is a mistake.

I first wrote some wrong advice here. So instead, I opened up the code base and fixed this issue.

That scope is not in use. And thus, should not be there, because [YAGNI](https://martinfowler.com/bliki/Yagni.html). And our enums follow a convention of always being lower case, so that is changed as well. When you pull the upstream changes, this will resolve.

No need to the test the scope that is not in use :).

Reply:
Great! Thank you!